TUTORIAL-TOOL.md

```diff
@@ -295,38 +319,40 @@ This concludes the editing phase.
 
 - Place all of the indoor map files (the `main.json` file and the level GeoJSON file[s]) and place them in a directory (e.g. `~/my-indoor-map`)
 - Zip the directory
-- On OS X / Linux, you can do this via opening a terminal and entering the following commands:
+- On Mac OS X, right click on the folder and select Compress
+- On Windows, navigate to the directory containing your indoor map files
+  - Right click > New > Compressed (zipped) Folder and choose an appropriate name
+  - Drag the .json & .geojson files onto the .zip file
+- On Linux, you can do this via opening a terminal and entering the following commands:
```

Comment:
Suggestion: indent the tree options so it's more obvious that is a branching of the zipping step